examples/internal/integration/integration_test.go

```diff
@@ -32,7 +32,7 @@ import (
 )
 
 type errorBody struct {
-	Error   string        `json:"error"`
+	Message string        `json:"message"`
 	Code    int           `json:"code"`
 	Details []interface{} `json:"details"`
 }
```

Comment:
Could this type be a `status.Status`?

Reply:
I believe we've got a test case doing that already. I'll double-check and either remove this one or change it.

Reply:
👉 https://github.com/grpc-ecosystem/grpc-gateway/blob/e6d05361fee32e0f75b57de7b98b9455742384a3/examples/internal/integration/proto_error_test.go#L71 

Looks like we can remove this one, then. Unless I'm mistaking their intention.